spec/spec_helper.rb

```diff
@@ -3,6 +3,7 @@
 require 'bundler/setup'
 require 'qiniu'
 require 'rspec'
+require 'webmock'
```

Comment:
@songjiayang  I tried this at the very beginning, and then I found not all specs requires this but I had to disable it in each spec. And DSLs defined in `webmock/rspec` are not as powerful as native APIs, that's why I still use native APIs.